packaging/prerelease/build_keybase.sh

```diff
@@ -16,7 +16,7 @@ build="$current_date+$commit_short"
 keybase_build=${KEYBASE_BUILD:-$build}
 tags=${TAGS:-"prerelease production"}
 platform=${PLATFORM:-`uname`}
-ldflags="-X github.com/keybase/client/go/libkb.CustomBuild=$keybase_build"
+ldflags="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$keybase_build"
```

Comment:
Ah shoot, I screwed this up. This changed needed to be replicated for Linux at `packaging/client/build_binaries.sh`. Doing it now.